console.lua

```diff
@@ -187,6 +187,97 @@ commands = {{
+        -e : Edition to spawn jokers withz
+        -E : enable jokers spawned to have random editions
+    joker remove : Removes jokers from joker slots
+        Usage: joker remove {count} 
+        if no count is provided remove all jokers
+    ]],
+    exec = function(args, rawArgs, dp)
+        if G.STAGE ~= G.STAGES.RUN then
+            return "This command must be run during a run.", "ERROR"
+        end
+        local j_key = nil
+        local j_count = 1
+        -- local j_no_edition = true
+        local j_set = "Joker"
+        local j_edition = nil
+        local j_rarity = nil
+        local add_to_deck = true
+        local smods_create_card = false -- SMODS and SMODS.create_card 
+        -- I thought of making a full on arg parser for this command
+        -- that started as a meme so I could spawn 1000 copies of jimbo without using eval
+        local skipNextArg = false
+        print(args[1])
+        if args[1] == "add" then
+            local skipNextArg = true
+            for i,v in ipairs(args) do 
+                if skipNextArg then
+                    skipNextArg = false
+                    -- Don't parse args to other parm
+                elseif not (v:sub(1,1) == "-" ) then
+                    j_key =  (v:sub(1,2) == "j_" ) and v or ("j_"..v)
+                    if not G.P_CENTERS[j_key] then 
+                        return "Did not recognize joker name: "..j_key, "ERROR"
+                    end
+                else
+                    skipNextArg = true
+                    local param = v:sub(2,2)
+                    if param == "a" then 
+                        add_to_deck = false
+                        skipNextArg = false
+                    elseif param == "E" then 
+                        j_no_edition = false
+                        skipNextArg = false
+                    elseif param == "S" then
+                        smods_create_card = SMODS and SMODS.create_card
+                        skipNextArg = false
+                    elseif not args[i+1] then 
+                        return "Please specify a value for the argument"
+                    end
+                    local val = args[i+1] 
+                    if param == "c" then 
+                        j_count = tonumber(val) 
+                        if not j_count then
+                            return "Argument to count must be a number"
+                        end
+                    elseif param == "e" then j_edition = "e_"..val
+                    elseif param == "r" then 
+                        j_rarity = val
+                    -- elseif param == "S" then j_set = val --Bad Idea
+                    end
+                end 
+            end
+            j_rarity = smods_create_card and j_rarity or tonumber(j_rarity) 
+            local j_skip_zmaterialize = j_count > 5
+            for _=1,j_count do
+                local card = SMODS.create_card{key = j_key, set = j_set, rarity = j_rarity,no_e, j_no_edition}
```

Comment:
this j_no_edition doesn't do anything as it's not assigned to a key. Also you should probably give SMODS your edition in this case instead of using set_edition. We also seem to not check if SMODS.create_card exists.